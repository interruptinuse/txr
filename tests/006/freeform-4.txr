@(next :list '("1" "2" "3" "4"))
@(collect)
@ (all)
@X
@Y
@ (and)
@  (freeform 2)
@FF
@ (end)
@(end)
@(output)
@ (repeat)
###
FF:
@FF /FF

X, Y : @X @Y

@ (end)
@(end)
