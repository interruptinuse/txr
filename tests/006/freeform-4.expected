###
FF:
1
2
 /FF

X, Y : 1 2

###
FF:
3
4
 /FF

X, Y : 3 4

X[0]="1"
X[1]="3"
Y[0]="2"
Y[1]="4"
FF[0]="1\
2\
"
FF[1]="3\
4\
"
